src/tags/index.js

```diff
@@ -91,7 +91,6 @@ module.exports = function(fractal){
                         }
                     }
 
-                    // @todo prevent mutliple convert and merge attributes from Twig render method only.
```

Comment:
Why did you remove this comment?

Reply:
Major refactoring tasks like this shouldn't be put in code as todos, but recorded as an issue instead; either here in this repo or elsewhere in the backlog of a project that needs it.